Makefile

```diff
@@ -344,12 +344,15 @@ markdownLintImage=tmknom/markdownlint
 containerMarkdownLint=$(PROJECT_NAME)-markdownlint
 containerMarkdownLintFix=$(PROJECT_NAME)-markdownlint-fix
 
+golangci_lint_cmd=go run github.com/golangci/golangci-lint/cmd/golangci-lint
+
+# NOTE: if you change the `lint` job then you must update ./github/workflows/lint.yml
```

Comment:
can we just use this command? this will fail so quickly and we will diverge

Reply:
we can - but this will lint "everything".